lib/spec_file_generator.js

```diff
@@ -70,10 +70,11 @@ function dependenciesToRequires(deps) {
       //  - just a version number, e.g. 1.2.3, meaning the requirement is equal to that
       //  - a constraint (<,>,>=,<=) and a version number, which can be used as-is
       versionDNF[0].forEach((constraint) => {
+        var m;
         if (constraint === '*') {
           dependencies.push('npm(' + dep + ')');
-        } else if (constraint.startsWith('>') || constraint.startsWith('<')) {
-          dependencies.push('npm(' + dep + ') ' + constraint);
+        } else if (m = constraint.match(/^(<=|>=|<|>|=)(.*?)$/)) {
```

Comment:
I think this is the same regex: `/^([<>]=?)(.*)?$/`

Reply:
I'm not sure if something like `=1.2.3` could reach that code point (which would need a space after `=`).  I'll try to find a real-life example.

Reply:
Nope, looking at [the specification for `package.json`](https://docs.npmjs.com/files/package.json) and [`semver`](https://docs.npmjs.com/misc/semver), you're right.  I'll amend the pull request.